gosrc/path.go

```diff
@@ -57,3 +57,9 @@ func IsGoRepoPath(path string) bool {
 func IsValidPath(importPath string) bool {
 	return pathFlags[importPath]&packagePath != 0 || IsValidRemotePath(importPath)
 }
+
+// IsStdlibPath returns true if path is in the standard library or $GOROOT
```

Comment:
Use IsGoRepoPath instead of adding this function.  pathFlags["C"] is set, but "C" is not in the standard library.